orchestra/utils/tests/test_task_lifecycle.py

```diff
@@ -181,9 +181,12 @@ def test_worker_assigned_to_rejected_task(self):
             task__status=Task.Status.POST_REVIEW_PROCESSING)
         self.assertTrue(assignments.exists())
         self.assertTrue(worker_assigned_to_rejected_task(self.workers[4]))
-        with self.assertRaises(TaskAssignmentError):
-            get_new_task_assignment(self.workers[4],
-                                    Task.Status.AWAITING_PROCESSING)
+        with patch('orchestra.utils.task_lifecycle.settings.'
```

Comment:
why was this new patch necessary?

Reply:
This test was failing when I ran locally because this setting was false. Insurance against flakiness.